### PR TITLE
Finished

### PR DESCRIPTION
**Pull request by sergiomb2**
Today , is what I did but still need some work

### CODE REVIEW

**Comment by kwizart** [comment]
NAK
If you really want to do rpmfusion-cert, please do start another project.
I don't think we need to do this for the long run. Better to not use fedora_cert at all

File: src/rpmfusion_cert/__init__.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def read_user_cert():
 
     """
     if os.path.exists(os.path.expanduser('~/.fedora.upn')):
-        with file(os.path.expanduser('~/.fedora.upn'), 'r') as f:
+        with open(os.path.expanduser('~/.fedora.upn'), 'r') as f:
```
Comment (kwizart) [inline src/rpmfusion_cert/__init__.py]: This change alone is acceptable.

**Comment by sergiomb2** [comment]
all fedora/rpmfusion_cert commands are needed at the moment `--new-cert` `--verify-cert` and get user from the cert , why you keep saying that is not need ? 
Moreover with remove of .fedora_cert , rfpkg is broken 
and moreover these 2 files are copied from fedora-packager, so make sense have the same in rpmfusion-packager , why a separate package ?

**Comment by sergiomb2** [comment]
> Better to not use fedora_cert at all

when fas certificate expires , people complains that can't build packages , so certificate still needed , do you plan migrate Infrastructure to Kerberos ? 

I already decide, I'm going build these tools on copr. Am I allowed to change license to GPLv3 like fedora-packager ?

**Pull request merged**